scamp/client.go

```diff
@@ -107,6 +110,26 @@ func (client *Client) Close() {
 		return
 	}
 
+	timeoutPeriod := 30 * time.Second
```

Comment:
I believe the default timeout when shutting down a docker container is ten seconds. If we're going to add a 30-second timeout we need to make sure we configure Kubernetes to allow for it.

Reply:
@aceeri Was this PR tested in your dev environment? We should test existing services like SDK to see if they behave correctly. We'll likely have to update SDK and ship-module to remove some of the shutdown logic in those services now that it's been moved to scamp-go. 

I originally thought we'd leave the detection of OS.Signal in the modules themselves and merely update service.Stop() to ensure everything shuts down gracefully when called, I'm not opposed to moving that logic within scamp-go but it somewhat limits things if a future service needs a longer shutdown period for example.

Reply:
I can remove the os.Signal from the announcer and we can take the timeout variable as a parameter.

Reply:
Ok let's do that, and we can have the timeout default to 10 seconds if it's not explicitly set.